plugins/optimization-detective/class-od-html-tag-walker.php

```diff
@@ -380,6 +408,21 @@ public function append_body_html( string $html ): bool {
 		return $success;
 	}
 
+	/**
+	 * Returns the uppercase name of the matched tag.
+	 *
+	 * This is a wrapper around the underlying WP_HTML_Tag_Processor method of the same name since only a limited number of
+	 * methods can be exposed to prevent moving the pointer in such a way as the breadcrumb calculation is invalidated.
+	 *
+	 * @since n.e.x.t
+	 * @see WP_HTML_Tag_Processor::get_tag()
+	 *
+	 * @return string|null Name of currently matched tag in input HTML, or `null` if none found.
+	 */
+	public function get_tag(): ?string {
+		return $this->processor->get_tag();
+	}
```

Comment:
This ensures that when passing the `OD_HTML_Tag_Walker` instance around, the current tag can always be obtained even when not looping over `open_tags()`.